python/py-aiohttp/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-aiohttp
+version             3.3.2
+categories-append   devel
+platforms           darwin
+license             Apache-2
+
+python.versions     35 36
+
+maintainers         {ipglider.org:miguel @ipglider} openmaintainer
+
+description         Async http client/server framework
+long_description    aiohttp key futures are the support for both client and \
+                    server side of HTTP protocol, support for both client and \
+                    server Web-Sockets out-of-the-box without the Callback Hell \
+                    and Web-server's middlewares and pluggable routing.
+
+homepage            https://github.com/aio-libs/aiohttp
+master_sites        pypi:a/aiohttp
+
+distname            aiohttp-${version}
+
+checksums           rmd160  bd2de873cd93a99aacf73280c2bab4130235b8ee \
+                    sha256  f20deec7a3fbaec7b5eb7ad99878427ad2ee4cc16a46732b705e8121cbb3cc12 \
+                    size    771414
+
+if {${name} ne ${subport}} {
+    depends_build       port:py${python.version}-setuptools
```

Comment:
Change this to `depends_build-append` to not overwrite the python portgroup dependencies.